scripts/build.js

```diff
@@ -0,0 +1,81 @@
+"use strict";
+var childProcess = require('child_process');
+var fs = require('fs');
+var os = require("os");
+var path = require("path");
+
+// Remove first two parameters which are node executable and this javascript file.
+var params = process.argv.slice(2);
+var clean = params.indexOf('clean') != -1 ? true : false;
```

Comment:
>params.indexOf('clean') != -1 ? true : false; [](start = 12, length = 45)

why not just "var clean = params.indexOf('clean') != -1" here?

Reply:
right, thanks.

---
In reply to: [134074858](https://github.com/Microsoft/napajs/pull/47#discussion_r134074858) [](ancestors = 134074858)